2020.11 (#3048)
biocore/qiita: CHANGELOG.md

# Qiita changelog

Version 2020.11
---------------

> * Deprecated the automatic creation of the per template QIIME1 mapping file. Now, if you want to merge a preparation and a biom table you must create a meta-analysis.

Reviewer: ```suggestion
* Deprecated the automatic creation of the per template QIIME1 mapping file. Iif you want to merge a preparation and a BIOM table you must first create a meta-analysis.
```

* Added a new autoloaded boolean flag to the Study object so we can control if a study was autoloaded via an automatic EBI-ENA or SRA loading job.

> * We stopped adding the CRC32 information from mod_zip, which should remove the warnings about CRC checksums.

Reviewer: ```suggestion
* We stopped adding the CRC32 information from mod_zip, which should remove the warnings about incorrect CRC checksums.
```

* Removed the show/hide button from an analysis while the analysis was being built to avoid possible confusion.
* A new per-sample, fast, bowtie2 and Woltka plugin for WGS and Metatrascriptomics processing has been added, deprecating the Shogun plugin and moving the rest of the commands to a new qp-meta plugin.

> * Added the possibility for plugins to submit and control their own jobs vs. Qiita automatially submitting for them. For specifics of this new functionality, visit (#3040)

Reviewer: ```suggestion
* Added the possibility for plugins to submit and control their own jobs vs. Qiita automatially submitting for them. For specifics of this new functionality, visit (![#3040](https://github.com/biocore/qiita/issues/3040))
```

> * We raised the number of workers in qiita.ucsd.edu for the web interface (from eight to twenty), redbiom (from eight to ten), and the plugin interactions (from eight to twenty). This should speed up responses and improve general performance.

Reviewer: ```suggestion
* We increased the number of workers in qiita.ucsd.edu for the web interface (from eight to twenty), redbiom (from eight to ten), and the plugin interactions (from eight to twenty). This should speed up responses and improve general performance.
```
